pkgs/development/python-modules/clickable/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, cookiecutter
+, argcomplete
+, pyyaml
+, jsonschema }:
+
+buildPythonPackage rec {
+  pname = "clickable";
+  version = "7.4.0";
+
+  src = fetchPypi {
+    pname = "clickable-ut";
+    inherit version;
+    sha256 = "sha256-xdRYPOsNX75pSh/IhClIpQ6zfijsKEY3FC5S5tZvEmY=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    cookiecutter
+    argcomplete
+    pyyaml
+    jsonschema
+  ];
+
+  doCheck = false;
```

Comment:
Can we just add a comment on why the check fails and we don't want to do them ?
Just so for someone in case they wanted to enable them in the future.